src/tests/java/htsjdk/variant/vcf/VCFHeaderLineTranslatorUnitTest.java

```diff
@@ -0,0 +1,35 @@
+package htsjdk.variant.vcf;
+
+import htsjdk.variant.VariantBaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+public class VCFHeaderLineTranslatorUnitTest extends VariantBaseTest {
+
+    @Test
+    public void testParseVCF4HeaderLine() {
+        final Map<String,String> values = VCFHeaderLineTranslator.parseLine(VCFHeaderVersion.VCF4_2, "<ID=SnpCluster,Description=\"SNPs found in clusters\">", null);
+        Assert.assertEquals(values.size(), 2);
+        Assert.assertEquals(values.get("ID"), "SnpCluster");
+        Assert.assertEquals(values.get("Description"), "SNPs found in clusters");
+
+        final Map<String,String> values2 = VCFHeaderLineTranslator.parseLine(VCFHeaderVersion.VCF4_2, "<ID=ANNOTATION,Description=\"ANNOTATION != \\\"NA\\\" || ANNOTATION <= 0.01\">", null);
+        Assert.assertEquals(values2.size(), 2);
+        Assert.assertEquals(values2.get("ID"), "ANNOTATION");
+        Assert.assertEquals(values2.get("Description"), "ANNOTATION != \"NA\" || ANNOTATION <= 0.01");
+
+        final Map<String,String> values3 = VCFHeaderLineTranslator.parseLine(VCFHeaderVersion.VCF4_2, "<ID=ANNOTATION,Description=\"ANNOTATION \\\\= \\\"NA\\\" || ANNOTATION <= 0.01\">", null);
+        Assert.assertEquals(values3.size(), 2);
+        Assert.assertEquals(values3.get("ID"), "ANNOTATION");
+        Assert.assertEquals(values3.get("Description"), "ANNOTATION \\= \"NA\" || ANNOTATION <= 0.01");
+
+        final Map<String,String> values4 = VCFHeaderLineTranslator.parseLine(VCFHeaderVersion.VCF4_2, "<ID=ANNOTATION,Description=\"ANNOTATION \\\\= \\\"NA\\\" || ANNOTATION <= 0.01\", Description2=\"foo\\\"bar\">", null);
+        Assert.assertEquals(values4.size(), 3);
+        Assert.assertEquals(values4.get("ID"), "ANNOTATION");
+        Assert.assertEquals(values4.get("Description"), "ANNOTATION \\= \"NA\" || ANNOTATION <= 0.01");
+        Assert.assertEquals(values4.get("Description2"), "foo\"bar");
+
```

Comment:
Add test case(s) for `\` before chars other than `\` or `"`, after patching the code to no longer escape other chars.